R/nest.R

```diff
@@ -101,9 +101,9 @@ nest <- function(.data, ..., .names_sep = NULL, .key = deprecated()) {
 
     cols <- c(cols_good, cols_fixed)
 
-    warn(paste0(
-      "All elements of `...` must be named.\n",
-      "Did you want `", .key, " = ", cols_fixed_label, "`?"
+    warn(c(
+      "All elements of `...` must be named.",
+      i = cli::format_inline("Did you want `{(.key)} = {cols_fixed_label}`?")
```

Comment:
This would avoid needing `format_inline()` here and below.

Reply:
Nice!